src/mono/mono.proj

```diff
@@ -505,6 +506,7 @@
     <ItemGroup>
       <FilesToStrip Include="$(_MonoRuntimeFilePath)" />
       <FilesToStrip Include="$([System.IO.Directory]::GetParent($(_MonoRuntimeFilePath)))\libmono-component-*$(SharedLibExt)" />
+      <FilesToStrip Include="$([System.IO.Directory]::GetParent($(_MonoRuntimeFilePath)))\Mono*framework\**\Mono*" Exclude="$([System.IO.Directory]::GetParent($(_MonoRuntimeFilePath)))\Mono*framework\**\*.dwarf" />
```

Comment:
Out of curiosity, why do we strip all of the mono files here, and why exclude the `.dwarf` file?

Reply:
We strip everything because that's what we do in dotnet - strip all libs and executables into .dwarf (Mac) and .dbg (Linux) files, which get included into the .symbols.nupkg packages.

They're excluded here, because on subsequent runs it causes `strip` to try and strip the .dwarf files themselves, which throws an error.